### PR TITLE
Fix on_ready() comment

### PR DESCRIPTION
**Pull request by HyperGH**
## Summary

`on_ready()` is called every time the bot connects to Discord, which can happen either when the bot starts up, or if it loses connection to Discord and reconnects, thus this statement is false.

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)

### CODE REVIEW

**Review by Lulalaby — DISMISSED** [review]

File: examples/views/button_roles.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ async def post(self, ctx: commands.Context):
 
     @commands.Cog.listener()
     async def on_ready(self):
-        """This function is called every time the bot restarts.
+        """This function is called every time the bot connects to Discord.
```
Comment (BobDotCom) [inline examples/views/button_roles.py:90]:
```suggestion
        """This function is called whenever the bot starts up.
```
"connects" isn't correct here, that's the connect event.

**Review by BobDotCom — DISMISSED** [review]

File: examples/views/button_roles.py (hunk 1 above)
Comment (Dorukyum) [reply examples/views/button_roles.py:90]: What about something like "whenever the internal cache and the connection of the bot is ready."?

Comment (HyperGH) [reply examples/views/button_roles.py:90]: That sounds more likey `is_ready()`,  so I would change it to "whenever the internal cache and the connection of the bot **becomes** ready."

Comment (Dorukyum) [reply examples/views/button_roles.py:90]: Sounds good to me.

Comment (Middledot) [reply examples/views/button_roles.py:90]: Maybe "This function is called every time the bot has a proper connection with Discord." or "... the bot has connected to Discord."

**Review by Middledot — APPROVED** [review]

File: examples/views/button_roles.py [hunk 2]
```diff
@@ -87,8 +87,8 @@ async def post(self, ctx: commands.Context):
 
     @commands.Cog.listener()
     async def on_ready(self):
-        """This function is called every time the bot restarts.
-        If a view was already created before (with the same custom IDs for buttons)
+        """This function is called whenever the internal cache and the connection of the bot
+        becomes ready. If a view was already created before (with the same custom IDs for buttons)
```
Comment (Dorukyum) [inline examples/views/button_roles.py:91]:
```suggestion
        become ready. If a view was already created before (with the same custom IDs for buttons)
```

**Comment by Lulalaby** [comment]
Fix merge conflicts phrase

**Pull request closed without merging**